### PR TITLE
UI profile other user

### PR DESCRIPTION
**Pull request by Keith-Go**
## Ticket
- [UI profile other user](https://trello.com/c/wwN3cpKp/31-ui-profile-other-user)

## Video

https://github.com/luke-nguyen-goldenowl/join_me/assets/152830147/6145cafb-1c52-4c80-9c5b-fa3991291a3f

### CODE REVIEW

File: lib/src/features/profile_other_user/logic/profile_other_user_bloc.dart [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import 'package:myapp/src/features/profile_other_user/logic/profile_other_user_state.dart';
+import 'package:myapp/src/network/domain_manager.dart';
+import 'package:myapp/src/network/model/common/pagination/pagination.dart';
+import 'package:myapp/src/network/model/event/event.dart';
+
+class ProfileOtherUserBloc extends Cubit<ProfileOtherUserState> {
```
Comment (luke-nguyen-goldenowl) [inline lib/src/features/profile_other_user/logic/profile_other_user_bloc.dart]: update with pagination bloc

File: lib/src/features/profile_other_user/logic/profile_other_user_state.dart [hunk 2]
```diff
@@ -0,0 +1,49 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
+import 'package:myapp/src/network/model/common/pagination/pagination.dart';
+import 'package:myapp/src/network/model/event/event.dart';
+
+class ProfileOtherUserState {
```
Comment (luke-nguyen-goldenowl) [inline lib/src/features/profile_other_user/logic/profile_other_user_state.dart]: pagination state

File: lib/src/features/profile_other_user/widget/tab_view_profile_user.dart [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+class TabViewProfileUser extends StatelessWidget {
+  const TabViewProfileUser({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<ProfileOtherUserBloc, ProfileOtherUserState>(
+      buildWhen: (previous, current) => previous != current,
```
Comment (luke-nguyen-goldenowl) [inline lib/src/features/profile_other_user/widget/tab_view_profile_user.dart]: update buildWhen

**Comment by Keith-Go** [comment]
resolved

**Comment by luke-nguyen-goldenowl** [comment]
Approved

**Pull request merged**